docs/cmdline.rst

```diff
@@ -0,0 +1,11 @@
+.. toctree::
+  :maxdepth: 2
+
+Command line options
+======================
+
+When invoking microraiden from the commandline, you have several options:
+
+.. click:: microraiden.click_helpers:main
```

Comment:
Docs show:
`microraiden [OPTIONS] COMMAND [ARGS]...` and 
`microraiden start [OPTIONS]`

Not valid commands at this point.